resources/pods.go

```diff
@@ -0,0 +1,39 @@
+package resources
```

Comment:
Filename was pluralized. I don't see a problem with that, but it should be consistent between resources.